Bender.yml

```diff
@@ -19,13 +19,14 @@ package:
     - Matheus Cavalcante <matheusd@iis.ee.ethz.ch>
 
 dependencies:
-  axi:                { git: https://github.com/pulp-platform/axi,                version:  0.39.0  }
+  axi:                { git: https://github.com/pulp-platform/axi,                rev:      snitch_future-tbenz }
   axi_riscv_atomics:  { git: https://github.com/pulp-platform/axi_riscv_atomics,  version:  0.6.0   }
   common_cells:       { git: https://github.com/pulp-platform/common_cells,       version:  1.28.0  }
   FPnew:              { git: "https://github.com/pulp-platform/cvfpu.git",        rev:      pulp-v0.1.3 }
   register_interface: { git: https://github.com/pulp-platform/register_interface, version:  0.4.2   }
   tech_cells_generic: { git: https://github.com/pulp-platform/tech_cells_generic, version:  0.2.11  }
   riscv-dbg:          { git: https://github.com/pulp-platform/riscv-dbg,          version:  0.8.0   }
+  idma:               { git: https://github.com/pulp-platform/iDMA,               rev:      __deploy__5c3eacc__dev-tbenz }
```

Comment:
This can be done, but maybe there is a solution that preserves versioning and provides the option of conflict control (to be discussed).

Reply:
Updated iDMA.